src/scripts/domain/moveCalculator.ts

```
@@ -238,7 +240,7 @@ export class MoveCalculator {
     pos: Vector2,
     direction: RVector2,
   ) {
-    if (!this.kingHasAttacksAlongLine(board, piece, toRemove, pos, direction)) {
+    if (!this.kingHasAttacksAlongLine(board, piece, pos, direction)) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)